src/main/java/school/faang/user_service/dto/mentorship/MentorshipRequestDto.java

```diff
@@ -1,17 +1,33 @@
 package school.faang.user_service.dto.mentorship;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.time.LocalDateTime;
 
 @Data
 @Builder
+@AllArgsConstructor
+@NoArgsConstructor
 public class MentorshipRequestDto {
-
     private Long id;
+
+    @NotEmpty(message = "The description should not be empty")
+    @NotBlank(message = "The description should not be blank")
+    @Size(min = 1, max = 4096, message = "The description must be between 1 and 4096 characters")
     private String description;
+
+    @NotNull(message = "The requester's ID must not be null")
```

Comment:
Тоже а вдруг empty?  Я тут статью нашла: https://www.baeldung.com/java-bean-validation-not-null-empty-blank И вот из нее цитата: 
@NotNull: a constrained CharSequence, Collection, Map, or Array is valid as long as it’s not null, but it can be empty.
@NotEmpty: a constrained CharSequence, Collection, Map, or Array is valid as long as it’s not null, and its size/length is greater than zero.
@NotBlank: a constrained String is valid as long as it’s not null, and the trimmed length is greater than zero.

Reply:
да, но не в случае числовых переменных ))

Reply:
принято:)